gunicorn/workers/ggevent.py

```diff
@@ -213,7 +213,7 @@ def log_request(self):
         resp_headers = getattr(self, 'response_headers', {})
         resp = GeventResponse(self.status, resp_headers, self.response_length)
         if hasattr(self, 'headers'):
-            req_headers = [h.split(":", 1) for h in self.headers.headers]
+            req_headers = self.headers.items()
         else:
             req_headers = []
```

Comment:
Shouldn't this be changed to `{}`?

Reply:
Nope. It's been a list of tuples and that isn't being changed here.